synapse/telepath.py

```diff
@@ -200,6 +237,33 @@ async def fini():
 
     return fini
 
+@contextlib.asynccontextmanager
+async def loadTeleCell(dirn):
+
+    certpath = s_common.genpath(dirn, 'certs')
+    confpath = s_common.genpath(dirn, 'cell.yaml')
+
+    usecerts = os.path.isdir(certpath)
+
+    ahaurl = None
+    if os.path.isfile(confpath):
+        conf = s_common.yamlload(confpath)
+        ahaurl = conf.get('aha:registry')
```

Comment:
this feels awkward to be resolving a cell config value from the cell.yaml when the value may be provided for a host cell but from a different source.

Reply:
From discussion:
Record envar values to the cell.yaml when the value is not present.
Change precedence so that the cell.yaml wins over the envars [to allow programmatic changes for things like mirror configurations].

Reply:
We should log a warning message when that precedence causes a envar value to whiff being set.

Reply:
done! :)